docs/source/start.ipynb

```diff
@@ -333,6 +333,67 @@
     "df.ww"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Woodwork with a Series\n",
+    "\n",
+    "Woodwork also can be used to store typing information on a Series. There are two approaches for initializing Woodwork on a Series. If you have a Series that has a dtype that is compatible with the specified or inferred LogicalType, Woodwork can be initialized through the `ww` namespace, just as with DataFrames."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "series = pd.Series([1, 2, 3], dtype='Int64')\n",
+    "series.ww.init(logical_type='Integer')\n",
+    "series.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As with DataFrames, Woodwork provides several methods that can be used to update or change the typing information associated with the series. As an example, add a new semantic tag to the series."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "series.ww.add_semantic_tags('new_tag')\n",
+    "series.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see from the output above, the specified tag has been added to the semantic tags for the series.\n",
+    "\n",
+    "In cases where the LogicalType would require the Series dtype to change, a helper function `ww.init_series` must be used. This function will return a new series object with Woodwork initialized and the dtype of the series changed to match the dtype of the LogicalType.\n",
+    "\n",
+    "To demonstrate this, first create a Series, with a `string` dtype. Then, initialize a Woodwork series with a `Categorical` logical type using the `init_series` function. The series that is returned will have Woodwork initialized with the LogicalType set to `Categorical` as expected."
```

Comment:
might be good to remind users that the reason why init series is necessary is because `Categorical`'s physical type is `category`